jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -132,6 +132,7 @@ extern bool ecma_is_value_object (ecma_value_t) __attr_pure___;
 extern void ecma_check_value_type_is_spec_defined (ecma_value_t);
 
 extern ecma_value_t ecma_make_simple_value (const ecma_simple_value_t value) __attr_const___;
+extern ecma_value_t ecma_make_boolean_value (const bool boolean_value) __attr_const___;
```

Comment:
(bool) is enough
const is not needed for primitive types, and name is not used as well.